### PR TITLE
create the Enumerate

### PR DESCRIPTION
**Pull request by camilevahviraki**
Dear reviewer, 
In this project my partner @JauJau088  and I @camilevahviraki we:

- [X] Created a class MyList that has an instance variable @list.
In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.

- [X] Create a module MyEnumerable that implements the following methods 

    #all?
    #any?
    #filter

Your review will be very appreciated :pray:

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+# test cases
+p(list.all? { |e| e < 5 })
+p(list.all? { |e| e > 5 })
+p(list.any? { |e| e == 2 })
+p(list.any? { |e| e == 5 })
+# p(list.filter { |e| e.even? })
```
Comment (mahbubul14) [inline my_list.rb]: - [ ] Kindly remove the inline comments from your code [line19]. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+# Enumerate
+
+> In this project we are learning how to use a module inside a Ruby class. For this we create a class MyList and a module MyEnumerable. Module MyEnumerable implements a subset of the functionality of Enumerable mixin.
+
+## Getting Started
+
+Install the gem file by executing:
+
+    $ bundle install
+
```
Comment (mahbubul14) [inline README.md]: - [ ] In your `README` file, Kindly update the "Getting Started" section. This should be an example of how you may give instructions on setting up your project locally. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @camilevahviraki & @JauJau088 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Munsa1 — APPROVED** [review]
Hi @camilevahviraki  @JauJau088, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**